pkg/client/scout/os_metadata_windows.go

```diff
@@ -6,13 +6,18 @@ import (
 	"context"
 	"strings"
 
+	"golang.org/x/sys/windows"
+
 	"github.com/datawire/dlib/dexec"
 	"github.com/datawire/dlib/dlog"
 )
 
 func getOsMetadata(ctx context.Context) map[string]interface{} {
 	cmd := dexec.CommandContext(ctx, "systeminfo")
 	cmd.DisableLogging = true
+	cmd.SysProcAttr = &windows.SysProcAttr{
```

Comment:
Same here. Don't think the `systeminfo` starts additional processes, so I don't understand the need for a process group. On the contrary actually. Don't we want this to die if the parent dies?